app/Member.php

```diff
@@ -22,10 +22,20 @@ class Member extends Model
      *
      * @var array
      */
-    protected $fillable = array(
+    protected $visible = [
         'last_name',
         'first_name',
         'gender',
+        'college_id',
+        'course_id',
+        'start_year',
+        'end_year',
+        'college_name',
+        'course_name',
+        'full_name',
+    ];
+
+    protected $hidden = [
```

Comment:
I believe you either use `$visible` (a whitelist) or `$hidden` (a blacklist) - it will automatically hide the rest / show the rest respectively.

Also, to clarify, the purpose of this is purely for serialisation to JSON.

Docs: https://laravel.com/docs/5.3/eloquent-serialization